src/Attribute/Connect.php

```diff
@@ -25,7 +27,7 @@ public function __construct(
 
     public function phpCode(): string
     {
-        return '$builder->connect(' . $this->varExport($this->getUri()) . ', ' . $this->varExport($this->getOptions()) . ');';
+        return '$builder->connect(' . $this->varExport($this->getUri()) . ', ' . $this->varExport(['controller' => $this->controller, 'action' => $this->action]) . ', ' . $this->varExport($this->getOptions()) . ');';
```

Comment:
i'm not convinced about this. please add tests with special options like `_ext` or `_method` (see https://book.cakephp.org/5/en/development/routing.html#reserved-route-elements)